packages/app/src/pages/PoolPage/index.tsx

```diff
@@ -1,35 +1,12 @@
-import cx from "classnames";
 import { BiDollarCircle } from "react-icons/bi";
-import { Outlet, useNavigate } from "react-router-dom";
+import { Outlet } from "react-router-dom";
 
-import { Button } from "~/components/Button";
 import { Card } from "~/components/Card";
 
-const style = {
-  button: `relative font-semibold text-xs`,
-};
-
-const SubpagesNav = () => {
```

Comment:
This needs to 
be a conditional thing, just removing makes impossible to arrive at the add liquidity page;

<img width="454" alt="Screen Shot 2022-05-25 at 8 41 57 PM" src="https://user-images.githubusercontent.com/3941923/170386985-8f798e89-b8f1-4118-a082-78713372d1e2.png">